### ISSUE TITLE
Search Filter - Is it possible to search for YouTube videos that has subtitles/CC ?

### ISSUE THREAD

**Issue by bekmuradov** [SuspiciousLookingOwl/youtubei#35]
In YouTube you can filter out the results based on subtitle feature.
Is it possible to add subtitle filter?

**Comment by SuspiciousLookingOwl** [comment]
I'm pretty sure it's possible.

However I still need to figure out how YouTube built their search filter query value, for example this url: https://www.youtube.com/results?search_query=example&sp=EgIYAw%253D%253D has 2 filters enabled, duration filter (4-20 minutes), and sort filter (by relevance), different filters combination resulting in different query value (`sp=???`), unlike how other websites usually handles it (e.g. `durationMin=4&durationMax=20&sortBy=relevance`)

Once I figured out how this work, it should be pretty easy to implement into the lib's API

**Comment by SuspiciousLookingOwl** [comment]
For now, I've added `params` option to the `.search()` method so you can pass raw params value (https://github.com/SuspiciousLookingOwl/youtubei/commit/2c21a0697e07761faf832e2047be87ee7b4c1ee5), for example if you want to filter out the results based on subtitle feature, it will look like this:
```
youtube.search("Keyword", {
    params: "EgIoAQ%253D%253D"
});
```
Keep in mind that `type` will be ignored if you passed `params`

Available in `0.0.1-rc.26`
